ReSharper.FSharp/src/FSharp/FSharp.Psi/src/FSharp.psi

```diff
@@ -745,6 +749,11 @@ lambdaExpr options { stubBase="FSharpExpressionBase"; } extras {
   lambdaParametersList<LAMBDA_PARAMETERS, Parameters>
   RARROW<RARROW, RArrow>
   fSharpExpression<EXPR, Expression>;
+
+dotLambdaExpr options { stubBase="FSharpExpressionBase"; }:
+  wildPat<UNDERSCORE, Underscore>
```

Comment:
I think it's not good to treat it as a pattern, even though it's the easiest way to support it. It just doesn't fit it properly, if we think about all the features and PSI code where this node may be used. I think we need a special case expression for it, like proposed here: https://github.com/dotnet/fsharp/pull/15548.